notebooks/features/cognitive_services/CognitiveServices - OpenAI.ipynb

```diff
@@ -0,0 +1,846 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Cognitive Services - OpenAI"
+   ],
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "title": "",
+     "showTitle": false,
+     "inputWidgets": null,
+     "nuid": "443f201a-bde9-4b71-80c2-ca05dfa6f36b"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Large language models are capable of successfully completing multiple downstream tasks with little training data required from users. This is because these models are already trained using enormous amounts of text. The 175 billion-parameter GPT-3 model for example, can generate text and even code given a short prompt containing instructions. \n\nWhile large models are becoming more powerful, more multimodal, and relatively cheaper to train, inferencing also needs to scale to handle larger volume of requests from customers. Using SynapseML, customers can now leverage enterprise grade models from Azure OpenAI Service to apply advanced language models on data stored in Azure Synapse Analytics. \n\nSynapseML is an open source library with a set of consistent APIs that integrate with a number of deep learning and data science tools, including Azure OpenAI. The OpenAI project itself maintains a [great tool](https://github.com/openai/openai-quickstart-node) for experimenting with GPT-3 to get an idea of how it works. SynapseML's integration with Azure OpenAI provides a simple and intuitive coding interface that can be called from Scala, Python or R. It is intended for use in industrial-grade applications, but it is also flexible enough to nimbly handle the demands of consumer website.\n\nThis tutorial walks you through a couple steps you need to perform to integrate Azure OpenAI Services to Azure SynapseML and how to apply the large language models available in Azure OpenAI at a distributed scale."
+   ],
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
```

Comment:
Should all the metadata sections be empty? I noticed they are empty in the other notebooks.